openff/interchange/_tests/interoperability_tests/internal/test_gromacs.py

```diff
@@ -98,57 +98,6 @@ def test_vaccum_warning(self, sage):
         with pytest.warns(UserWarning, match="gitlab"):
             out.to_gro("tmp.gro")
 
-    @pytest.mark.slow
-    @skip_if_missing("openmm")
-    def test_residue_info(self, sage):
```

Comment:
FTR: Any removed tests are duplicated; some amount of this file is just leftover from a botched merge some time ago